packages/react-spinbutton/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@fluentui/react-spinbutton",
+  "version": "0.1.0",
+  "description": "Package to hold SpinButton using button v8",
+  "private": true,
+  "main": "lib/index.js",
```

Comment:
This isn't quite right--it looks like you chose the `node` template in `create-package` rather than `react`. It might be best to just rename the package folder to some temp name, re-run `create-package` with `react` template, copy over any files you manually modified, and delete the old version.